consensus/istanbul/core/commit_test.go

```diff
@@ -190,6 +231,15 @@ OUTER:
 			}
 		}
 
+		// core should have received a parent seal from each of its neighbours
+		// how can we add our signature to the ParentCommit? Broadcast to ourselve
```

Comment:
Good question, this is a tricky one. I *think* we don't need to do anything here, since if we were online and participating in consensus, we would have included our own commit in whatever block we wind up getting. So if, when proposing, we use the union of the commits we saw from others, and the commits we have in the previous block, we should get included by way of the latter.


@kevjue, is that your understanding as well?

Reply:
I agree that the union approach should fix this.

Reply:
I also agree with the approach.